Jenkinsfile

```diff
@@ -7,7 +7,19 @@ node {
 
   stage('Build') {
     // you should build this repo with a maven build step here
-    echo "hello"
-  }
+        withMaven (maven: maven3) {
+            sh "mvn package"    }
   // you should add a test report here
-}
```

Comment:
I think you deleted one too many `}`s

Reply:
I think I fixed this however I am still getting a build error now, not sure what is going on.